scripts/sysman/device.yml

```diff
@@ -130,6 +173,17 @@ members:
       name: "driverVersion[$S_STRING_PROPERTY_SIZE]"
       desc: "[out] Installed driver version (NULL terminated string value). Will be set to the string \"unkown\" if this cannot be determined for the device."
 --- #--------------------------------------------------------------------------
+type: struct
+desc: "Device properties"
+version: "1.7"
+class: $sDevice
+name: $s_device_ext_properties_t
+base: $s_base_properties_t
+members:
+    - type: $s_uuid_t
+      name: uuid
+      desc: "[out] universal unique identifier. Note: Subdevices will have their own uuid."
```

Comment:
should we add note here (and also maybe expand in Sysman Prog guide) that the `uuid` here would be equivalent to the uuid in Core device properties? in Prog guide we could explain with small pseudo code how user could correlate a ze_device_handle_t with zes_device_handle_t using the uuid..

Reply:
Yes, uuid is common between core and syman and can use that to check and operate in core and sysman device handle. Need to to add on the programing guide

Reply:
uuid obtained from Sysman API is the same as from core API